modules/twinklespeedy.js

```diff
@@ -1409,7 +1409,7 @@ Twinkle.speedy.callbacks = {
 				return;
 			}
 
-			var xfd = /(?:\{\{([rsaiftcm]fd|md1|proposed deletion)[^{}]*?\}\})/i.exec( text );
+			var xfd = /(?:\{\{((?:article for deletion|template for discussion)\/dated|[cfm]fd|proposed deletion).*?\}\})/i.exec( text ) || /(?:#invoke:(RfD))/.exec(text);
```

Comment:
I don't think it's necessary, but we could also insert `\b` as suggested in #391.  That is:

`/(?:\{\{((?:article for deletion|template for discussion|prod blp|proposed deletion)\/dated|[cfm]fd\b)[\s\S]*?\}\})/i`

That might help with the original issue aka any other templates that lead off with `[cfm]fd`.

Reply:
JS does have an s modifier. But if we use it, `.*?\}\}` part of the regex won't be of any use since it could match the end of _any_ template coming after the text matched so far. So I just removed that part and now in turn, adding the modifier is not necessary.

Is there any reason to wrap the whole regexes with `(?:   )` , which I have removed just now?

Reply:
I suppose not?  @MusikAnimal any thoughts?  I'm guessing it was all there to be as parsimonious as possible and avoid conflicts, with the advent of `/dated` and the `\b` perhaps none of that is really needed.

I _do_ thing the `proposed deletion` should be moved in front of the dated option to avoid matching other templates.